projects/storefrontstyles/scss/theme/sparta/_variables.scss

```diff
@@ -9,11 +9,13 @@
 //   1) Add new variables
 @import '../../cxbase/functions';
 
-//fonts (see _fonts.scss to import)
-$font-family-sans-serif: 'Open Sans', -apple-system, BlinkMacSystemFont,
-  'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji',
+$font-family: 'Open Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI',
+  'Roboto', 'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji',
   'Segoe UI Emoji', 'Segoe UI Symbol';
 
+//fonts (see _fonts.scss to import)
+$font-family-sans-serif: $font-family !default;
```

Comment:
is this necessary? If not, let's keep it as what it was before as I don't see a need to set it as a default from the issue at hand